cfn-resources/util/constants/constants.go

```diff
@@ -25,7 +25,6 @@ const (
 
 	CustomMasterKey        = "AwsKms.CustomerMasterKeyID"
 	RoleID                 = "AwsKms.RoleID"
-	Region                 = "AwsKms.Region"
```

Comment:
What's the basis for this removal?

Reply:
Region was not in use, hence removed unused constant